website/docs/docs/tutorials/typescript.md

```diff
@@ -168,26 +168,56 @@ more information.
 Once you have your model definition you can provide it as a type argument to the
 `createStore` function.
 
+Given that you have the following `./todos.model.ts`:
 ```typescript
-import { createStore, computed, action, thunk } from 'easy-peasy';
-import { StoreModel } from './model';
+import { action, Action, computed, Computed, thunk, Thunk } from 'easy-peasy';
 
-const store = createStore<StoreModel>({
-  todos: [],
-  completedTodos: computed((state) => state.todos.filter((todo) => todo.done)),
+interface Todo {
+  text: string;
+  done: boolean;
+}
+
+export interface c {
+  items: Todo[];
+  completedItems: Computed<TodosModel, Todo[]>;
+  addTodo: Action<TodosModel, Todo>;
+  saveTodo: Thunk<TodosModel, Todo>;
+}
+
+const todosStore: TodosModel = {
+  items: [],
+  completedItems: computed((state) => state.items.filter((todo) => todo.done)),
   addTodo: action((state, payload) => {
-    state.todos.push(payload);
+      state.todos.push(payload);
   }),
   saveTodo: thunk(async (actions, payload) => {
-    const result = await axios.post('/todos', payload);
-    actions.addTodo(result.data);
+      const result = await axios.post('/todos', payload);
+      actions.addTodo(result.data);
```

Comment:
thanks for doing this!

Can we rename `addTodo` and `saveTodo` to `addItem` and `saveItem` respectively?

Reply:
Hmmm, also thinking about it. It might be better to just use a flat store model in this example. Avoid the mental jump of having to understand nested models to begin with.

So rename `TodosModel` -> `StoreModel` and `items` -> `todos`.

Will maybe make it easier to understand.

We could consider adding a further section to discuss nested model structures.

Reply:
From what I have seen in the wild most uses of Easy Peasy are just flat stores.

Reply:
I think showing off the possibility of having nested stores is a good selling point. But I agree, for this scenario, it is probably best to keep it as simple as possible 👍